src/api/spec/models/user_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
-
+# rubocop:disable Metrics/BlockLength
```

Comment:
We usually don't do this, we normally add this file to the ignored files for that cop.

Reply:
ok, I added 
```
Metrics/BlockLength:
  Exclude:
    - "**/*_spec.rb"

```

and then rubocop complained that we have alread it on:
```
src/api/spec/controllers/webui/package_controller_spec.rb:# rubocop:disable Metrics/BlockLength
src/api/spec/controllers/webui/project_controller_spec.rb:# rubocop:disable Metrics/BlockLength
```

thats why I did it online, but I can put it global as I did. It make sense for Rspec which are mainly blocks

Reply:
@mdeniz 

> We usually don't do this, we normally add this file to the ignored files for that cop.

We actually do it... It is in several places in the tests... but we shouldn't :see_no_evil: :joy:

Reply:
```
grep -r "# rubocop:disable " * | wc -l
74
```

Reply:
I know this exists, but we had agreed in not doing this anymore...